config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes-sigs/node-feature-discovery:
+  - name: pull-node-feature-discovery-verify
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: verify
+      description: "Verify source code of node-feature-discovery: check formatting, lint etc."
+    spec:
+      containers:
+      - image: golang:1.13
```

Comment:
now that we have a PR for bumping to 1.14, we should also bump it here

Reply:
Good point! Now updated to 1.14